src/triedbmut.rs

```diff
@@ -20,6 +20,7 @@ use crate::triedb::TrieDB;
 use crate::{Trie, TrieError, TrieMut};
 use ccrypto::{blake256, BLAKE_NULL_RLP};
 use cdb::{DBValue, HashDB};
+use lru_cache::LruCache;
```

Comment:
There is no change in performance when writing the value to the trie, so I will exclude the cache from this.